src/Cohesity.Powershell/Scripts/Vlan/Update-CohesityVlan.ps1

```diff
@@ -0,0 +1,84 @@
+function Update-CohesityVlan {
+    <#
+        .SYNOPSIS
+        Updates the vlan.
+        .DESCRIPTION
+        The Update-CohesityVlan function is used to update vlan.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Update-CohesityVlan  -VlanId 18 -Subnet 10.18.144.0 -NetmaskBitsForSubnet 20 -Gateway 10.18.144.1
```

Comment:
Designed for piping ?

Reply:
Fixed : Piping has been introduced for update operation.